script.js

```diff
@@ -1,4 +1,77 @@
+var SCISSORS = "scissors";
+var PAPER = "paper";
+var STONE = "stone";
+var REVERSE_SCISSORS = "reversed scissors";
+var REVERSE_PAPER = "reversed paper";
+var REVERSE_STONE = "reversed stone";
+var replayInstruction = "Type scissor, paper, or stone to play another round!";
```

Comment:
Very, very nice use of global variables! This allows us to do comparisons like `(input == SCISSORS)` instead of repeatedly saying `(input == "scissors")`.

Also, storing these strings as variables will allow you to make heavy changes across the code by only changing the strings stored in said variables once. Hypothetically, if the English dictionary writers decide one day that "scissors" should be spelt "scizzors" instead, you need only change line 1 to `var SCISSORS = "scizzors";`. This change will be applied throughout your code without having to do a Ctrl/Cmd+F for the string "scissors".

The only small thing I will add is regarding the consistency of your variable naming. It is indeed a convention to use uppercase characters for global variables, like `var SCISSORS ...`. However, `replayInstruction` is currently in camel case. It will be great to keep this global variable naming convention consistent, by changing the variable name to `REPLAY_INSTRUCTION` instead, like so `var REPLACE_INSTRUCTION = ....`